src/ziggurat/mapper.clj

```diff
@@ -9,12 +9,12 @@
             [ziggurat.sentry :refer [sentry-reporter]])
   (:import (java.time Instant)))
 
-(defn- send-msg-to-channel [channels message-payload return-code]
+(defn- send-msg-to-channel [channels message-payload return-code headers]
```

Comment:
are we using these headers in Rabbitmq? or are we just persisting them in Rabbit?
If its just persisting why not add headers to the message-payload?

Reply:
these are used by the tracing connection to track the trace headers and has to passed in as properties

Reply:
okay.